_docs/quickstart.md

```diff
@@ -124,25 +126,21 @@ $
 ```
 
 To get help on any `flux` subcommand or API program, the `flux help`
-command may be used. For example, to view the man page for the `flux-up(1)`
+command may be used. For example, to view the man page for the `flux-hwloc(1)`
 command, use
 
 ```console
-$ flux help up
+$ flux help hwloc
 ```
 
 ### Interacting with a Flux session
 
 There are several low-level commands of interest to interact with
-a Flux sessions. For example, to view states of broker ranks within
-the current session, `flux up` may be used:
+a Flux sessions. For example, to view system topology from each flux-broker and query the resulting data stored in the Flux Key Value Store, `flux hwloc` may be used:
 
 ```console
-$ flux up
-ok:     [0-3]
-slow:   
-fail:   
-unknown:
+$ flux hwloc info
+3 Machines, 108 Cores, 108 PUs
```

Comment:
It is strange that only 3 Machines are listed here even though the `srun` command above launched a 4 node instance.

Reply:
Ah, you're so right. When I was generating the new output for the examples, I launched a 3-node instance instead of a 4 node instance... I'm sorry for not paying closer attention to that!